### PR TITLE
median of the two sorted arrays

### LINKED ISSUE #4787: Median of Two Sorted Arrays
## 💥 Proposal

I am a GSOC participant and want to work on Median of Two Sorted Arrays using javascript

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

### PR DESCRIPTION
**Pull request by Amit366**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

(Write your answer here.)
Given two sorted arrays nums1 and nums2 of size m and n respectively, return the median of the two sorted arrays.

## Related Issues or Pull Requests

Fixes: #4787

### CODE REVIEW

**Comment by Amit366** [comment]
@HarshCasper please review this PR.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Few changes: 

- Make Markdown changes 
- Don't use prompts
- Don't put the code in `Miscellaneous`. We will delete it soon. Put it in JS

**Comment by Amit366** [comment]
@HarshCasper should I take input ?? 
And also what is markdown changes??

**Comment by Amit366** [comment]
@HarshCasper please check it now

File: JavaScript/cp/Median_of_Two_Sorted_Arrays.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Space complexity: O(n)
+
+Example 1:
+
+Input:
+[1,3]
+[2]
+
+Output:
+2.00000
```
Comment (HarshCasper) [inline JavaScript/cp/Median_of_Two_Sorted_Arrays.js:51]: I don't think that the I/O format is proper

File: JavaScript/cp/Median_of_Two_Sorted_Arrays.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+/*Given two sorted arrays nums1 and nums2 of size m and n 
+respectively, return the median of the two sorted arrays.*/
+ 
+ // Prompt node package for taking user input
+const prompt = require("prompt-sync")({ sigint: true });
+
+
+var findMedianSortedArrays = function(nums1, nums2) {
+    //concatenate and sort the arrays
+  let nums3 = nums1.concat(nums2).sort(function(a, b){return a-b});
```
Comment (HarshCasper) [inline JavaScript/cp/Median_of_Two_Sorted_Arrays.js:11]: Use meaningful variable names

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by Amit366** [comment]
@HarshCasper I am closing this as I lost this branch on my vscode. I have created a new PR .please check that

**Pull request closed without merging**